source/features/show-names.tsx

```diff
@@ -78,6 +78,7 @@ function init(signal: AbortSignal): void {
 	observe(usernameLinksSelector, batchUpdateLinks, {signal});
 }
 
+void features.addCssFeature(import.meta.url);
```

Comment:
This should only be used for CSS-only features. If there's already an `init` function we can add the class as `document.body.classList.add(something)`

Reply:
This applies to all

Reply:
Oops, missed the other one, thanks!